l10n/en-US/viewer.properties

```diff
@@ -265,3 +265,8 @@ editor_free_text_font_color=Font Color
 editor_free_text_font_size=Font Size
 editor_ink_line_color=Line Color
 editor_ink_line_thickness=Line Thickness
+
+# Editor aria
+editor_free_text_aria_label=FreeText Editor
+editor_ink_aria_label=Ink Editor
+editor_ink_canvas_aria_label=User created image
```

Comment:
Shouldn't this be `User-created image` according to hyphen rules in English (looking at https://www.grammarbook.com/punctuation/hyphens.asp and https://www.grammarly.com/blog/hyphen)?